app/views/schools/participants/show.html.erb

```diff
@@ -63,6 +63,16 @@
           <% end %>
         </dd>
       </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Start date
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @induction_record&.start_date.to_date.to_s(:govuk) %>
```

Comment:
Just want to check that this is the correct date they are expecting here - this gets set on every change to an induction record?